bootstrap.sh

```diff
@@ -37,12 +35,17 @@ if [ "$1" == "--force" ]; then
     rm -rf ~/Library/Caches/org.carthage.CarthageKit
 fi
 
-# Import locales
-if [ -d "/firefoxios-l10n" ] && [ -d "/ios-l10n-scripts" ]; then
-    echo "l10n directories found. Not downloading scripts."
-else
-    echo "l10n directory not found. Downloading repo and scripts."
-    getLocale
+if [ "$1" == "--importLocales" ]; then
```

Comment:
@boek How would this work with out `--locale`, is this something we should document in our readme? 🤔